src/_utils/themeProvider/index.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { ReactNode } from 'react'
+import { createGlobalStyle } from 'styled-components'
+
+import { color } from '../branding'
+
+const CDN_BASE_URL = 'https://cdn.blablacar.com/kairos/assets/fonts'
```

Comment:
should go to global ../branding file since we could also use other assents from there.
Also this is FONT_CDN_BASE_URL

Reply:
Not completely convinced (I like to keep things local as long as possible, here it's only used in this file).
But it's easy to move around so I moved it for now.